### ISSUE TITLE
Bug in display Tagify inside input

### ISSUE THREAD

**Issue by vikingbr** [yairEO/tagify#307]
Hi,

Sorry for that Issue, is my first, but...
I need suggest two things:

1. I suggest for you change in your class **tagify** in file **tagify.css** the attribute display: 
`display: flex;` to `display: inline-table !important; (for override others class in input)`
 Because with `display flex`, the tags don't stay inside the input "_tagifyed_".

2. And... the input where JSON of TAGS is saved (apparently the input existed before "_tagifyed_", **i may be wrong**), stay displayed after "_tagify_", for me, it needs to be hidden after "_tagify_" the inpud..

Oficial edit 1.0:
May be is most pretty you remove `display: block`; in class **tagify input**. That enable the writing new tag in same line of before tags. Not in a new line, how is it done now.

Thank you for the project,
Very much success for you!

**Comment by yairEO** [comment]
Can you please make a drawing of your desired design?

I don't understand what are all those changes for, because it works flawlessly now.
Also, I don't understand how is it possible you are seeing the original input, unless you have styles of yourself being applied, with higher specificity.

Tagify's CSS rule for hiding the original input is:

```css
.tagify+input, .tagify+textarea {
    border: 0;
    display: none;
}
```

**Comment by vikingbr** [comment]
Hi,
Thanks for the answer.

**About number 1:**
Yesterday, at night, i created a new project for test. Out of my production project, and the tagify worked very well. I really believe it is because some imported library that was overwriting the styles.

**About number 2:**
Yes, really after create issue, i see in my input the "form-control" instead, and after removal stopped displaying original input. I really apologize for that.


_In short the mistake was mine and create a "issue" because of this was hasty. Thank you for your attention._

**Very successful in your project, I will continue following!**

**Comment by yairEO** [comment]
Thank you very much!

I will release a new Tagify version tomorrow with many bugfixes and some updates
